test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DisasterRecovery/BackupTests.cs

```diff
@@ -63,17 +73,11 @@ public void CreateBackupTest()
             // Get default backup path
             BackupConfigInfo backupConfigInfo = DisasterRecoveryService.Instance.GetBackupConfigInfo(helper.DataContainer, sqlConn, sqlConn.Database);
             string backupPath = backupConfigInfo.DefaultBackupFolder + "\\" + databaseName + ".bak";
-            
-            var backupInfo = new BackupInfo();
-            backupInfo.BackupComponent = (int)BackupComponent.Database;
-            backupInfo.BackupDeviceType = (int)BackupDeviceType.Disk;
-            backupInfo.BackupPathDevices = new Dictionary<string, int>() { { backupPath, (int)DeviceType.File } };
-            backupInfo.BackupPathList = new List<string>(new string[] { backupPath });
-            backupInfo.BackupsetName = "default_backup";
-            backupInfo.BackupType = (int)BackupType.Full;
-            backupInfo.DatabaseName = databaseName;
-            backupInfo.SelectedFileGroup = null;
-            backupInfo.SelectedFiles = "";
+
+            BackupInfo backupInfo = createBackupInfo(databaseName,
+                BackupType.Full,
+                new List<string>(new string[] { backupPath }),
```

Comment:
>new string[] { backupPath  [](start = 33, length = 26)

why create an array and then throw it away to construct a list...seems like a minor abuse of the memory heap...I think "new List<string>() { backupPath };" works fine here.